universe/syncer.go

```diff
@@ -74,14 +74,39 @@ func (s *SimpleSyncer) executeSync(ctx context.Context, diffEngine DiffEngine,
 		s.isSyncing.Store(false)
 	}()
 
+	// If we're not syncing a specific set of Universes, then we'll
+	// sync all Universes which have an associated specific sync config.
+	// These universes are filtered based on the actual config content
+	// later.
+	if len(idsToSync) == 0 {
```

Comment:
I think for allowing opt-in global sync, we would still want the old logic that uses `RootNodes` (the old default case of the switch), but we could gate on having a global sync config with insert set to true.

Then switching default behavior from opt-out to opt-in should be just setting this flag to false?

https://github.com/lightninglabs/taproot-assets/blob/81eb8d72e75dd567ba4e2ff3f15a1226a2d20fdc/tapdb/universe_federation.go#L49

And we wouldn't need a new daemon flag.

Reply:
I think what you're suggesting will work. Is a user hasn't explicitly modified the global config in the db then they should still be using the default. And so modifying the default global config should affect most users when they upgrade.